### ISSUE TITLE
Variable of datetime type instead of string when format=post

### ISSUE THREAD

**Issue by javiquinte** [EIDA/routing#16]
`item['end']` is of type `datetime.datetime` and is supposed to be a `string`. The + operator with another string is throwing an error only in case that the format is `POST`. The query to test it is:
```
http://rz-vm258.gfz-potsdam.de/eidaws/routing/1/query?net=GE&format=post```

**Comment by javiquinte** [comment]
Fixed in commit 7f6e4f0c933fcc47ea03487ca566813a2a157647

**Comment by javiquinte** [comment]
If the `end` parameter was given this error is still happening. At least with geofon-open1.

**Comment by javiquinte** [comment]
It seems now that is solved with commit c8b7b915da1dc0cde2d17cdd92b4eabe5ac2cc46
